spec/acceptance/attendances_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "rspec_api_documentation/dsl"
+
+resource "Attendances" do
+  header "Content-Type", "application/json"
+  let(:school) { create(:school) }
+  let!(:staff_1) { create(:staff_with_standards, school_id: school.id) }
+  let!(:staff_2) { create(:staff_with_standards, school_id: school.id) }
+
+  before do
+    SchoolTiming.days.keys.each do |day|
+      FactoryBot.create(:school_timing, school_id: school.id, day: day)
+    end
+  end
+
+  get "v1/attendances/sync", document: :v1 do
```

Comment:
Add specs for following scenario as well:
1. when date parameter not provided.
2. when school id not valid
3. when date parameter is invalid format
4. when start date is in  future.